code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -331,6 +331,25 @@ GLOBAL_LIST_EMPTY(bloodmen_list)
 	if(!isnull(lighting_alpha))
 		user.lighting_alpha = min(user.lighting_alpha, lighting_alpha)
 
+/obj/item/clothing/glasses/telepathy
+	name = "blindfold of telepathy"
+	desc = "Covers the eyes, preventing natural sight, but provides thought-sense to those who focus and stay dedicated"
```

Comment:
I am totally open to new descriptions btw, if anyone has any suggestions

Reply:
It's a good explanatory description but could use some more flavor text.

Reply:
the "those who focus etc" qualifier is a lie because all you have to do is wear it, and then dedication is compulsory until you get your head surgically removed and reattached
uhhh something similarly mystical, maybe like
_"In return for committing oneself forever to the senses of the mind, the senses of the body are allowed to rest."_
something that at least implies it's cursed